radix-engine/src/vm/vm.rs

```diff
@@ -33,7 +37,56 @@ impl<'g, W: WasmEngine, E: NativeVmExtension> Clone for Vm<'g, W, E> {
     }
 }
 
+/// Api provided to clients of the VM layer
+pub trait VmApi {
+    /// Retrieve the current minor version of the Scrypto VM
+    fn get_scrypto_minor_version(&self) -> u64;
+}
+
+/// Simple implementation of the VmAPI
+#[derive(Debug, Clone, Copy, Default)]
+pub struct VmVersion {
+    scrypto_v1_minor_version: u64,
+}
+
+impl VmApi for VmVersion {
+    fn get_scrypto_minor_version(&self) -> u64 {
+        self.scrypto_v1_minor_version
```

Comment:
Discussed this with Josh, this is something that we will need to change later once we start having multiple major versions of the Scrypto VM, this can be changed later with no issues.